src/domain/users/user-organizations.repository.ts

```diff
@@ -174,6 +170,19 @@ export class UsersOrganizationsRepository
     return invitations;
   }
 
+  private async createUserAndWallet(args: {
```

Comment:
Not strictly related, but I've extracted some code from `UsersOrganizationsRepository['inviteUsers']` as it's getting large and harder to read.